cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/passes/DFGPass.kt

```diff
@@ -138,6 +139,33 @@ class DFGPass(ctx: TranslationContext) : ComponentPass(ctx) {
         }
     }
 
+    /**
+     * Handle a [ThrowStatement]. Currently, we support two types of [ThrowStatement.exception],
+     * which are then forwarded to the appropriate handlers:
+     * - [CallExpression]
+     * - [Reference]
+     */
+    protected fun handleThrowStatement(
+        node: ThrowStatement,
+        inferDfgForUnresolvedSymbols: Boolean
+    ) {
+        node.exception?.let { exc ->
+            when (exc) {
+                is CallExpression -> {
+                    handleCallExpression(exc, inferDfgForUnresolvedSymbols)
```

Comment:
In fact, this should be much easier to handle ;) I guess what you want to do is to generate a DFG from the `node.exception` to `node` and maybe also from `node.cause` to `node`? This would be simply:
```kt
node.exception?.let{ node.prevDFGEdges += it }
node.cause?.let{ node.prevDFGEdges += it }
```
Since the exception and cause are part of the AST, we will visit and handle them automatically.

Reply:
Done